data/mods/MindOverMatter/recipes/practice/vitakinesis_practice.json

```diff
@@ -626,5 +627,56 @@
         }
       }
     ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "name": "contemplation: ameliorate extremity",
+    "id": "practice_vita_limb_restore",
+    "description": "Contemplate your powers and improve your ability to restore whole limbs, but at an extreme cost.",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "metaphysics",
+    "difficulty": 9,
+    "time": "1 h",
```

Comment:
I think the double dust cost is a good (and should probably be expanded to other level 10 powers), but the time should be the same as the other practice recipes at 30 minutes. It's all balanced around how much XP reading spellbooks gives in Magiclysm and that provides the same XP-per-second regardless of Difficulty.